src/sandbox-client.ts

```diff
@@ -294,78 +354,57 @@ export class SandboxClient {
    * any sandbox/template created on codesandbox.io, even your own templates) or don't pass
    * in anything and we'll use the default universal template.
    *
-   * This function will also start & connect to the VM of the created sandbox, and return a {@link Sandbox}
-   * that allows you to control the VM.
+   * This function will also start & connect to the VM of the created sandbox as a ROOT session, and return a {@link Sandbox}
+   * that allows you to control the VM. Pass "autoConnect: false" to only return the session data.
    *
    * @param opts Additional options for creating the sandbox
    *
    * @returns A promise that resolves to a {@link Sandbox}, which you can use to control the VM
    */
   async create(
     opts: { autoConnect: false } & CreateSandboxOpts
-  ): Promise<SandboxStartData>;
+  ): Promise<SessionData>;
   async create(
     opts?: { autoConnect?: true } & CreateSandboxOpts
   ): Promise<Sandbox>;
   async create(opts?: CreateSandboxOpts): Promise<Sandbox>;
-  async create(opts?: CreateSandboxOpts): Promise<Sandbox | SandboxStartData> {
-    const templateId = opts?.template || this.defaultTemplate;
-    const privacy = opts?.privacy || "public";
-    const tags = opts?.tags || ["sdk"];
-    const path = opts?.path || "/SDK";
-
-    // Always add the "sdk" tag to the sandbox, this is used to identify sandboxes created by the SDK.
-    const tagsWithSdk = tags.includes("sdk") ? tags : [...tags, "sdk"];
-
-    const result = await sandboxFork({
-      client: this.apiClient,
-      body: {
-        privacy: privacyToNumber(privacy),
-        title: opts?.title,
-        description: opts?.description,
-        tags: tagsWithSdk,
-        path,
-        start_options:
-          opts?.autoConnect === false
-            ? undefined
-            : startOptionsFromOpts(opts || {}),
-      },
-      path: {
-        id: typeof templateId === "string" ? templateId : templateId.id,
-      },
-    });
-
-    const sandbox = handleResponse(result, "Failed to create sandbox");
-
-    return this.connectToSandbox(sandbox.id, () => {
-      if (sandbox.start_response) {
-        return Promise.resolve(sandbox.start_response);
-      }
+  async create(opts?: CreateSandboxOpts): Promise<Sandbox | SessionData> {
+    // We always want to start the VM in this context as our intention it to connect immediately,
+    // or return the session data to manually connect, for example in browser
+    const sandbox = await createSandbox(this.apiClient, true, opts);
+    const shouldReturnSessionOnly = opts?.autoConnect === false;
+    const session: SessionData = {
+      id: sandbox.id,
+      pitcher_token: sandbox.start_response.pitcher_token,
+      pitcher_url: sandbox.start_response.pitcher_url,
+      user_workspace_path: sandbox.start_response.user_workspace_path,
+    };
 
-      return this.start(sandbox.id, opts);
```

Comment:
I do not quite get this. If you pass `autoConnect: false` it will NOT give `start_response` as you are not passing `start_options`, but you will still connect to the sandbox, just start it more slowly? I assume it was a boo boo and fixed it 😄